### PR TITLE
Add tizen arm64 rootfs build script

### PR DESCRIPTION
**Pull request by gbalykov**
cc @alpencolt

### CODE REVIEW

File: eng/common/cross/arm64/tizen-fetch.sh [hunk 1]
```diff
@@ -0,0 +1,170 @@
...
+}
+
+Inform "Initialize arm base"
+fetch_tizen_pkgs_init standard base
+Inform "fetch common packages"
+fetch_tizen_pkgs aarch64 gcc glibc glibc-devel libicu libicu-devel libatomic linux-glibc-devel
+Inform "fetch coreclr packages"
+fetch_tizen_pkgs aarch64 lldb lldb-devel libgcc libstdc++ libstdc++-devel libunwind libunwind-devel lttng-ust-devel lttng-ust userspace-rcu-devel userspace-rcu
+Inform "fetch corefx packages"
+fetch_tizen_pkgs aarch64 libcom_err libcom_err-devel zlib zlib-devel libopenssl libopenssl1.1-devel krb5 krb5-devel libcurl libcurl-devel
```
Comment (jkotas) [inline eng/common/cross/arm64/tizen-fetch.sh:164]: libcurl and libcurl-devel should not be needed anymore

File: eng/common/cross/arm64/tizen-fetch.sh [hunk 2]
```diff
@@ -0,0 +1,170 @@
...
+		fi
+	done
+}
+
+Inform "Initialize arm base"
+fetch_tizen_pkgs_init standard base
+Inform "fetch common packages"
+fetch_tizen_pkgs aarch64 gcc glibc glibc-devel libicu libicu-devel libatomic linux-glibc-devel
+Inform "fetch coreclr packages"
+fetch_tizen_pkgs aarch64 lldb lldb-devel libgcc libstdc++ libstdc++-devel libunwind libunwind-devel lttng-ust-devel lttng-ust userspace-rcu-devel userspace-rcu
```
Comment (jkotas) [inline eng/common/cross/arm64/tizen-fetch.sh:162]: Why do we need rcu?

Comment (gbalykov) [reply eng/common/cross/arm64/tizen-fetch.sh:162]: lltng requires rcu:
```
/home/z/Dev/runtime/.tools/rootfs/armel/usr/include/lttng/tracepoint-rcu.h:26 fatal error: 'urcu/compiler.h' file not found
  #include <urcu/compiler.h>
           ^~~~~~~~~~~~~~~~~
  :10: fatal error: 'urcu/compiler.h' file not found
```

File: eng/common/cross/arm64/tizen-fetch.sh (hunk 1 above)
Comment (gbalykov) [reply eng/common/cross/arm64/tizen-fetch.sh]: Fixed

**Comment by jkotas** [comment]
The new scripts need to be include here https://github.com/dotnet/arcade/blob/9131f071c1b48da0f35466e632a212913f21a3f0/eng/configure-toolset.sh#L8 to suppress the CI error

**Comment by gbalykov** [comment]
@jkotas I've fixed CI, could you, please, take a look

**Pull request merged**